system/modules/core/library/Contao/Model.php

```diff
@@ -891,13 +937,22 @@ public static function findOneBy($strColumn, $varValue, array $arrOptions=array(
 	 */
 	public static function findBy($strColumn, $varValue, array $arrOptions=array())
 	{
+		$blnModel = false;
+
+		$arrColumn = (array) $strColumn;
```

Comment:
I may be a string or an array in your implementation but was only a string before.
Otherwise the check on `count($arrColumn) == 1` is not needed as it will always be 1.